ubuntu-dev/ubuntu-dev.dockerfile

```diff
@@ -141,6 +141,16 @@ RUN git clone https://github.com/ninja-build/ninja /tmp/ninja \
  && echo ". /home/user/.ninja-bash-completion" >> /home/user/.bashrc \
  && rm -rf /tmp/ninja
 
+# Install the latest watchman.
+RUN git clone https://github.com/facebook/watchman.git /tmp/watchman \
```

Comment:
Nit: `.git` suffix isn't really needed here.

Reply:
Well, it was a git convention until GitHub made it smart through backend code. I'd rather keep the suffix.